api/base/filters.py

```diff
@@ -173,6 +177,8 @@ def param_queryset(self, query_params, default_queryset):
             for field_name, value in fields_dict.items():
                 if self.is_filterable_field(key=field_name):
                     queryset = queryset.intersection(set(self.get_filtered_queryset(field_name, value, default_queryset)))
+                else:
+                    raise InvalidFilter()
```

Comment:
Yeah, I always thought was weird that you could raise classes rather than error instances. I don't feel strongly about it though, so I can change it.